tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
I mean that a mod without initialization script could inherit it from a parent mod. That's what this intends to avoid.